packages/documentation/.storybook/preview.js

```diff
@@ -20,6 +22,23 @@ export const parameters = {
     },
   },
   docs: {
+    theme: postTheme,
+    components: {
+      // Remove default storybook styles from most of things (helps with dark mode in mdx files)
+      h1: styled.h1(() => {}),
+      h2: styled.h2(() => {}),
+      h3: styled.h3(() => {}),
+      h4: styled.h4(() => {}),
+      h5: styled.h5(() => {}),
+      h6: styled.h6(() => {}),
+      p: styled.p(() => {}),
+      ul: styled.ul(() => {}),
+      li: styled.li(() => {}),
+      dl: styled.dl(() => {}),
+      dt: styled.dt(() => {}),
+      dd: styled.dd(() => {}),
+      code: styled.code(() => {}),
```

Comment:
```suggestion
```

This line causes problems with the normal Source Code preview.
And we're overriding it anyway...